replica/database.cc

```diff
@@ -1026,11 +1026,11 @@ future<> database::add_column_family(keyspace& ks, schema_ptr schema, column_fam
         throw std::invalid_argument("Column family " + schema->cf_name() + " exists");
     }
     ks.add_or_update_column_family(schema);
-    cf->start();
     schema->registry_entry()->set_table(cf->weak_from_this());
     auto holder = co_await _cf_lock.hold_write_lock();
     _column_families.emplace(uuid, std::move(cf));
     _ks_cf_to_uuid.emplace(std::move(kscf), uuid);
+    _column_families[uuid]->start();
```

Comment:
you can get the iterator to this entry as the result of `_column_families.emplace` above to save an extra search of uuid.